typing_extensions/src_py3/typing_extensions.py

```diff
@@ -503,6 +504,142 @@ class Other(Leaf):  # Error reported by type checker
     return f
 
 
+if hasattr(typing, 'Literal'):
+    Literal = typing.Literal 
+elif sys.version_info[:2] >= (3, 7):
+    class _LiteralForm(typing._SpecialForm, _root=True):
+
+        def __repr__(self):
+            return 'typing_extensions.' + self._name
+
+        def __getitem__(self, parameters):
+            return _GenericAlias(self, parameters)
```

Comment:
Logically this is more like `_VariadicGenericAlias`, but since we don't add any special meaning for `...` (Ellipsis), this is OK.